tracer/src/Datadog.Trace/AppSec/AsmRemoteConfigurationProducts.cs

```diff
@@ -16,4 +16,6 @@ internal static class AsmRemoteConfigurationProducts
     public static AsmDataProduct AsmDataProduct { get; } = new();
 
     public static AsmDDProduct AsmDDProduct { get; } = new();
+
+    public static AsmProduct AsmProduct { get; } = new();
```

Comment:
I hope _we_ didn't choose these names, because what on earth is the difference between `AsmProduct`, `AsmDDProduct`, `AsmDataProduct` 😂

Reply:
That is something I don't understand, also. They come from the RemoteConfig specs :_